app/views/govuk_publishing_components/components/_intervention.html.erb

```diff
@@ -22,7 +26,7 @@
     <p class="govuk-body">
       <%= tag.span suggestion_text, class: "gem-c-intervention__textwrapper" if suggestion_text %>
       <% if suggestion_link_text and suggestion_link_url %>
-        <%= tag.a suggestion_link_text, class: "govuk-link gem-c-intervention__suggestion-link", href: suggestion_link_url, data: suggestion_data_attributes %>
+        <%= tag.a suggestion_link_text, class: "govuk-link gem-c-intervention__suggestion-link", href: suggestion_link_url, data: suggestion_data_attributes, target: target %>
```

Comment:
Could we add a `rel` attribute here to account for [security risks that come with opening links in new tabs](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)?

```suggestion
        <%= tag.a suggestion_link_text, class: "govuk-link gem-c-intervention__suggestion-link", href: suggestion_link_url, data: suggestion_data_attributes, target: target, rel: "noopener noreferrer" %>
```

Reply:
We'd also want to add `external` to that rel attribute if the new tab was indeed an external site. I'm not sure how best to check for that though...

Reply:
👍

Reply:
[Done](https://github.com/alphagov/govuk_publishing_components/pull/2465/commits/5b436da4299f4c0026fdf7e19cafdf85a0885a2b)